packages/elements-next/src/docz/docs-wrapper/docs-wrapper.tsx

```diff
@@ -29,7 +30,7 @@ export const DocsWrapper: React.FC<DocsWrapperProps> = ({ children }) => {
         <CopyToClipboard text={markup} onCopy={handleClipboardCopy(setIsCopied)}>
           <BsClipboard />
         </CopyToClipboard>
-        <Highlight {...defaultProps} code={markup} theme={github} language="jsx">
+        <Highlight {...defaultProps} code={jsbeautify.html(markup)} theme={github} language="markup">
```

Comment:
Can you try to import `prettier` and use it instead, I think it also supports HTML